modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationSpi.java

```diff
@@ -403,6 +401,50 @@
     /** Handshake wait message type. */
     public static final short HANDSHAKE_WAIT_MSG_TYPE = -28;
 
+    /** Communication metrics group name. */
+    public static final String COMMUNICATION_METRICS_GROUP_NAME = "communication.tcp";
```

Comment:
Let's use `MetricUtils#metricName` instead of `"communication.tcp"`.

Reply:
ok